app/views/citationInfo.html

```diff
@@ -62,12 +62,15 @@ <h2 class="no-indent">Ticket Info</h2>
               <span ng-if="!ctrl.isMissingFineInfo(ctrl.selectedCitation)">
                 ${{ctrl.getFineTotal(ctrl.selectedCitation)}}
               </span>
-              <span id="fine-warning" ng-if="!ctrl.isMissingFineInfo(ctrl.selectedCitation)">
-                Note: this is the amount of your fine only.  Your total will also include court fees.
-              </span>
               <span class="italic" ng-if="ctrl.isMissingFineInfo(ctrl.selectedCitation)">
                 Fine Info Unavailable
               </span>
+              <span id="fine-warning" class="citation-info-warning-note" ng-if="!ctrl.isMissingFineInfo(ctrl.selectedCitation) && !ctrl.isMissingFineInfo(ctrl.selectedCitation)">
```

Comment:
Your expression has the same two arguments: `!ctrl.isMissingFineInfo(ctrl.selectedCitation)` - please remove one.